c2rust-analyze/tests/filecheck/known_fn.rs

```diff
@@ -0,0 +1,19 @@
+use std::ffi::{c_char, c_int};
+
+extern "C" {
+    fn access(
+        path: *const c_char,
+        amode: c_int,
+    ) -> c_int;
+}
+
+// CHECK-LABEL: final labeling for "known_fn"
+pub fn known_fn() {
```

Comment:
this looks great, are there some more examples you can add here that appear in lighttpd?

Reply:
Yeah, there'll definitely be more.  How many is useful to add here?  Are there any particular cases that you think would be helpful to test?